Editors/vscode/package.json

```diff
@@ -68,6 +68,14 @@
                         "verbose"
                     ],
                     "description": "Traces the communication between VS Code and the SourceKit-LSP language server."
+                },
+                "sourcekit-lsp.scheme": {
+                    "type": "array",
+                    "default": [],
+                    "items": {
+                        "type": "string"
+                    },
+                    "description": "Scheme to pass to sourceit-lsp, index visibility will be reduced to the selected targets"
```

Comment:
+1, we may have other uses for passing a restricted list of targets, it's advantageous to make the extension specific to its purpose.

Reply:
To make the api explicit, it would require the IDE to have the knowledge of the whole build graph, which feels more like build graph's responsibility rather than IDE to me.

Reply:
The IDE is likely going to be in direct communication with the build system, so the IDE retrieving a set of targets from the build system based on certain criteria and passing them along to SKLSP seems like a plausible scenario.